src/main/java/org/nuxeo/segment/io/listener/SegmentIOAsyncListener.java

```diff
@@ -23,16 +23,12 @@
 
 public class SegmentIOAsyncListener implements PostCommitEventListener {
 
-    protected static Log log = LogFactory.getLog(SegmentIOAsyncListener.class);
-
-    protected SegmentIOComponent getComponent() {
-        return (SegmentIOComponent) Framework.getRuntime().getComponent(SegmentIOComponent.class.getName());
-    }
+    private static Log log = LogFactory.getLog(SegmentIOAsyncListener.class);
 
     @Override
     public void handleEvent(EventBundle bundle) throws ClientException {
 
-        SegmentIOComponent component = getComponent();
+        SegmentIOComponent component = (SegmentIOComponent) Framework.getService(SegmentIO.class);
```

Comment:
Weird to call a service a component.
And I think there is a slight difference between the two concepts.